### PR TITLE
Improve Project Settings window size docs

### PR DESCRIPTION
**Pull request by ghost**
I tried to make it clear that `test_width` and `test_height` weren't just editor-only debug options, and are proper window size overrides. Sort of a stopgap until https://github.com/godotengine/godot/issues/16863#issuecomment-412308210 happens. Please review since I'm not entirely sure about the wording.

### CODE REVIEW

**Comment by akien-mga** [comment]
Thanks!

File: doc/classes/ProjectSettings.xml [hunk 1]
```diff
@@ -403,19 +403,19 @@
 			Sets the window to full screen when it starts.
 		</member>
 		<member name="display/window/size/height" type="int" setter="" getter="" default="600">
-			Sets the main window height. On desktop, this is the default window size. Stretch mode settings use this also as a reference when enabled.
+			Sets the game's main viewport height. On desktop platforms, this is the default window size. Stretch mode settings also use also as a reference when enabled.
```
Comment (Anutrix) [inline doc/classes/ProjectSettings.xml]: `also use also` should be `also use this`

Comment (homer666) [reply doc/classes/ProjectSettings.xml]: Wow, can't believe I didn't pick up on that. Fixed by #32555.

Comment (Anutrix) [reply doc/classes/ProjectSettings.xml]: Great

**Pull request merged**